### PR TITLE
feat(dop): sceneSet name case sensitive

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
sceneSet name not case sensitive

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb25JRHMiOlsxMTkwXSwiYXNzaWduZWUiOlsiMTAwMTI2MSJdfQ%3D%3D&id=307079&iterationID=1190&pId=0&type=REQUIREMENT)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    sceneSet name case sensitive          |
| 🇨🇳 中文    |      场景集名称区分大小写        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/dop/services/sceneset/sceneset.go [hunk 1]
```diff
@@ -67,6 +63,24 @@ func (svc *Service) CreateSceneSet(req apistructs.SceneSetRequest) (uint64, erro
 	return sceneSet.ID, nil
 }
 
+func (svc *Service) sceneSetNameCheck(spaceID uint64, name string, setID uint64) bool {
```
Comment (sfwn) [inline modules/dop/services/sceneset/sceneset.go:66]: you'd better to declare the items to check.
like `sceneSetCaseSensitiveNameCheck`

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/dop/services/sceneset/sceneset.go (hunk 1 above)
Comment (littlejiancc) [reply modules/dop/services/sceneset/sceneset.go]: good idea

**Review by sfwn — APPROVED** [review]

File: modules/dop/dao/sceneset.go [hunk 2]
```diff
@@ -52,6 +52,12 @@ func (client *DBClient) CountSceneSetByName(name string, spaceId uint64) (int, e
 	return res, nil
 }
 
+func (client *DBClient) FindSceneSetsByName(name string, spaceId uint64) ([]SceneSet, error) {
+	var res []SceneSet
+	err := client.Model(&SceneSet{}).Where("`space_id` = ? and name = ?", spaceId, name).Find(&res).Error
```
Comment (sfwn) [inline modules/dop/dao/sceneset.go:57]: add backquote for `name` field.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/dop/dao/sceneset.go (hunk 2 above)
Comment (littlejiancc) [reply modules/dop/dao/sceneset.go]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**